airflow/www/templates/airflow/dags.html

```diff
@@ -215,6 +215,7 @@ <h2>DAGs</h2>
   <script>
 
       const DAGS_INDEX = "{{ url_for('Airflow.index') }}";
+      const DAGS_AUTOCOMPLETE = "{{ url_for('Airflow.dags_autocomplete') }}";
```

Comment:
I know about the existence of an amendment, but this change has not been accepted yet, so I follow the old way.
https://github.com/apache/airflow/pull/4787/files
Depending on which change will be accepted first, the relevant PR will update.